src/lib/setup.js

```diff
@@ -62,6 +61,7 @@ const {
   COOKIE_DOMAIN,
   DEFAULT_CACHE_TIME,
   IP_DENYLIST,
+  // This value may not match process.env.NODE_ENV
```

Comment:
Mind explaining more in the comment?

Reply:
Oh, it was a message to remind me to check it. It wasn't clear that the config and env vars would always align. I'm am semi-confident that this is always the case. I would love to find time to clean up how config + env is loaded.